src/Orchard.Web/Modules/Orchard.Blogs/Migrations.cs

```diff
@@ -39,8 +39,8 @@ public class Migrations : DataMigrationImpl {
                     .WithPart("AutoroutePart", builder => builder
                         .WithSetting("AutorouteSettings.AllowCustomPattern", "True")
                         .WithSetting("AutorouteSettings.AutomaticAdjustmentOnEdit", "False")
-                        .WithSetting("AutorouteSettings.PatternDefinitions", "[{\"Name\":\"Title\",\"Pattern\":\"{Content.Slug}\",\"Description\":\"my-blog\"}]")
-                        .WithSetting("AutorouteSettings.DefaultPatternIndex", "0"))
+                        .WithSetting("AutorouteSettings.PatternDefinitions", "[{\"Name\":\"Title\",\"Pattern\":\"{Content.Slug}\",\"Description\":\"my-blog\",\"Culture\":\"\"}]")

```

Comment:
Since I have now the option to enable/disable the culture patterns ; I just added the current patterns to the default culture leaving the other cultures with the default pattern "{Content.Slug}". So, by default the culture patterns will never be enabled. And after the migration it will also be set to false. So, it should not break any actual patterns. Though, if you say that you want me to CLONE those old patterns into every cultures already installed we could do that also in the migration. But there is also the fact that we enable the culture patterns for making a distinction between each culture patterns. So they will need to edit/save them anyhow to do that.